app/models/procedure.rb

```diff
@@ -694,6 +694,11 @@ def before_publish
 
   def after_publish(canonical_procedure = nil)
     update!(published_at: Time.zone.now, canonical_procedure: canonical_procedure, draft_revision: create_new_revision, published_revision: draft_revision)
+    published_revision.update!(published_at: Time.zone.now)
```

Comment:
Pinaillage :
```suggestion
    published_revision.touch(:published_at)
```
?

(ça va aussi modifier le `updated_at` en plus du `published_at`, mais j'ai l'impression que c'est ce qu'on veut.)